pkgs/development/compilers/graalvm/graalvm-oracle/hashes.nix

```diff
@@ -0,0 +1,38 @@
+{
+  "22" = {
+    "aarch64-linux" = {
+      sha256 = "sha256-0wPTDBB2T+qrB27+eQGd88mBsEh/ysPfelMT929hBA4=";
+      url = "https://download.oracle.com/graalvm/22/archive/graalvm-jdk-22_linux-aarch64_bin.tar.gz";
```

Comment:
Those URLe are unacceptable, especially in this case since Hydra will not cache the result (because this is unfree).

We need either to have a stable URL from upstream or have the files archived somewhere (e.g.: archive.org).

Reply:
there are some precedents, but they typically result in the packages being marked as broken (see fmod)

Reply:
I found more stable links (with full version strings) [here](https://www.oracle.com/java/technologies/javase/graalvm-jdk22-archive-downloads.html).

I'm not sure if this is sufficient from what you're saying. This package will always be unfree, regardless where we download it from? And I'm not sure it's legal to replicate it to archive.org.

Reply:
the ones with full version strings should be sufficient